pages/settings.php

```diff
@@ -69,6 +69,69 @@ function userSettings() {
             $system->message($e->getMessage());
         }
         $system->printMessage();
+    }
+	else if (!empty($_POST['upload_avatar'])) {
+		$valid_file_types = ['gif', 'png', 'jpg'];
+        try {
+			$target_dir = __DIR__ . '/../_user_imgs/';
+			$target_file = $target_dir . $system->db->clean(basename($_FILES['fileToUpload']['name']));
+			$file_type = strtolower(pathinfo($target_file, PATHINFO_EXTENSION));
+			$target_file_path = $target_dir . strtolower($player->user_name) . '.' . $file_type;
+			$user_link = $system->router->base_url . '_user_imgs/' . strtolower($player->user_name) . '.' . $file_type;
+			$current_file_path = null;
+
+			foreach($valid_file_types as $img_type) {
+				if(file_exists($target_dir . strtolower($player->user_name) . '.' . $img_type)) {
+					$current_file_path = $target_dir . strtolower($player->user_name) . '.' . $img_type;
+					break;
+				}
+			}
+			
+			// Limit uploads to 1/day
+			if($current_file_path && (time() - filemtime($current_file_path)) < 86400) {
```

Comment:
```suggestion
			if($current_file_path && (time() - filemtime($current_file_path)) < 60) {
```
Idk that this will create a ton of load, let's be more permissive here